### ISSUE TITLE
Editing colours?

### ISSUE THREAD

**Issue by nordicsocialist** [alienator88/slacky-dark#7]
Hey man!

First of forgive me for opening an issue, but you did not provide an email so I figured what the hell hehe.

I've been ripping my eyes out because of the light theme for too long and found your amazing one just recently. However I would like to edit sidebar on the desktop app (Switching in preferences does nothing?)

And I would also like to edit the message text colour in the channels / DM's without it affecting any other text (like username different colours, sidebar texts). 

Is that possible?

![image](https://user-images.githubusercontent.com/45404614/51618156-08511900-1f2e-11e9-9aa9-8779906634b3.png)

To clarify I want to edit the blue arrow text ,but leave the red one intact.

**Comment by alienator88** [comment]
Hey, no problem man.

So those colors can't be changed directly in my theme from the settings because the theme style overwrites the custom colors set in the app settings. So changing those won't make a difference.

If you wanted to edit the sidebar and text color, you could do that if you forked my repo and made your own changes to the dark.css file to replace the colors.

**Comment by nordicsocialist** [comment]
Yeah that's what I thought I had to do, I played around with it but from all the hundreds CSS syntaxes I couldn't find the right one. You don't have to have which lines / syntax affect that text?

**Comment by alienator88** [comment]
For the font, try changing the html color code for this element:
.c-message__body { color: #e6e6e6; }

For the sidebar, there's a few different ones that make up the whole sidebar:

Main sidebar color: .p-channel_sidebar__list { background-color: #2d2d2d; }
Top section with name/workspace info: #team_menu { background: #2d2d2d; }

Hope that helps!

**Comment by nordicsocialist** [comment]
Amazing man, thanks a ton! 

Do you have the syntax for the channel colours too? Both read and unread.

**Comment by nordicsocialist** [comment]
Seems no dice about the sidebar, I added: 


.p-channel_sidebar__channel.p-channel_sidebar__channel--im.p-channel_sidebar__channel--unread { color: #26ff00; }

But still stays the same colours. Also tried changing every single #e6e6e6 to another colour across the CSS, which changed everything except sidebar text. Weird...

**Comment by alienator88** [comment]
Hey, sorry I was a bit busy yesterday. 
So just to confirm, are you trying to change the background color of the sidebar or the text color?

The dark gray in the main chat window should be #222222 and the lighter gray is #2D2D2D

**Comment by nordicsocialist** [comment]
Hey!

I am trying to change both just to play around abit.

I'd like to edit:

The background of the channel window / PM text window. 
The background of the sidebar
The channel names / text colour of the sidebar
The unread channel names / text colour of the sidebar.

**Comment by nordicsocialist** [comment]
That's totally fine. If you ever get the time to look at it, just shoot a message here :) 

Thank you for your time and good luck with exams!
